lang/src/org/partiql/lang/eval/physical/PhysicalPlanCompilerImpl.kt

```diff
@@ -1847,6 +1848,23 @@ internal class PhysicalPlanCompilerImpl(
         }
     }
 
+    private fun compilePivot(expr: PartiqlPhysical.Expr.Pivot, metas: MetaContainer): PhysicalPlanThunk {
+        val inputBExpr: RelationThunkEnv = bexperConverter.convert(expr.input)
+        val keyExpr = compileAstExpr(expr.key)
+        val valueExpr = compileAstExpr(expr.value)
+        return thunkFactory.thunkEnv(metas) { env ->
+            val attributes: Sequence<ExprValue> = sequence {
+                val relation = inputBExpr(env)
+                while (relation.nextRow()) {
+                    val key = keyExpr.invoke(env)
+                    val value = valueExpr.invoke(env)
+                    yield(key.namedValue(value))
```

Comment:
Per Spec Section 14:
```
PIVOT v AT a
....
The expression a is expected to evaluate into a string value. In strict mode, it is an error if this evaluates to a
non-string value. In permissive mode, the attribute is considered MISSING and does not appear in the output. The
expression v can be any PartiQL value, but if it is MISSING it will not be generated in the resulting tuple.
```

Consider: 
`PIVOT x.v AT i FROM [ { 'a' : 'first', 'v' : 'John' }, {'a' : 'last', 'v' : 'doe' } ] AS x AT i` 
Two things to note here: 
1. The current evaluating compiler returns an empty struct in `LEGACY` mode, which arguably should be an error. 
2. The current logic in the PR produces `{'null': 'John', 'null': 'doe'}`. 

Some additional logic may be need here, something like
```kotliin
if (key.type.isText) {
      val value = valueExpr.invoke(env)
      yield(key.namedValue(value))
}
else{
     // yield missing to conform with the current behavior
     yield(valueFactory.MissingValue)
}
```